tutorials/text/usetex.py

```diff
@@ -51,6 +51,32 @@
       "font.family": "serif",
       "font.serif": ["Palatino"],
   })
+  # It's also possible to use the reduced notation by directly setting font.family:
+  plt.rcParams.update({
+    "text.usetex": True,
+    "font.family": "Helvetica"
+  })
+
+Currently, the supported fonts are:
```

Comment:
I'm not sure what @tacaswell meant by listing fonts, but I think this list is incomplete. The fonts in the second column are just the default which Matplotlib tries. What's available depends on what's installed on the computer running the code.

Reply:
No, that's correct as the supported list *of tex fonts* (which should be available on any tex install that includes e.g. texlive-fonts-recommended).

Reply:
Ah right, forgot this is about `usetex` there.